### PR TITLE
Wishlist

### PR DESCRIPTION
**Pull request by neetukumari4858**
Feat :Added wishlist
On the product card-
1.I can remove the item from the wishlist.
2.Add the item to the cart.

### CODE REVIEW

File: Cart-wishlist/wishlist.html [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+                    <a  class="nav-link" href="/Product-list/product-list.html">Shop</a>
+                </ul>
+            </div>
+            <div class="nav-section">
+                <div class="outer-search-bar-div">
+                    <button class="btn"><img class="zoom" src="/Image/Zoom.png" height="38" width="32"></button>
+                    <input type="text" class="search-bar" placeholder="Search..">
+                </div>
+                <ul class="nav-icons">
+                    <i class='fas fa-user-alt' style='font-size:20px'>
```
Comment (Avin008) [inline Cart-wishlist/wishlist.html:30]: avoid inline-style

File: Cart-wishlist/wishlist.css [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+}
+.card-btn:hover{
+    background-color:var(--red-color);
+}
+.card-text{
+    text-align: center;
+    height: 3.3rem;
+    width: 18rem;
+}
+s{
```
Comment (Avin008) [inline Cart-wishlist/wishlist.css:79]: Which element are you referring to using s?

**Review by Avin008 — APPROVED** [review]
Everything looks clean. good to merge

File: Cart-wishlist/wishlist.html [hunk 3]
```diff
@@ -0,0 +1,136 @@
...
+                    <i class="fa fa-shopping-cart" style='font-size:20px'>
+                        <a class="nav-link icon-content" href="cart.html">Cart</a>
+                    </i>
+                </ul>
+            </div>
+        </nav>
+        <!--  First card -->
+        <div class="outer-product-side-bar">
+            <div class="product-container">
+                <h2>My<span>Wishlist</span> </h2>
```
Comment (fahd23) [inline Cart-wishlist/wishlist.html:45]: What does span do here, any styling purpose?

**Review by fahd23 — APPROVED** [review]
Looks good, you can merge it

File: Cart-wishlist/wishlist.html (hunk 3 above)
Comment (neetukumari4858) [reply Cart-wishlist/wishlist.html:45]: Here I have used span for inline elemnet.

File: Cart-wishlist/wishlist.html (hunk 1 above)
Comment (neetukumari4858) [reply Cart-wishlist/wishlist.html:30]: I have used Here s  tag <s>₹ 500</s>

File: Cart-wishlist/wishlist.css (hunk 2 above)
Comment (neetukumari4858) [reply Cart-wishlist/wishlist.css:79]: I have used here s  tag <s>₹ 500</s>

**Pull request merged**